File: Semester4/Advanced Programming/ThriftProj/JavaThrift/Lab2_Hw/Model/src/main/java/domain/exceptions/ValidatorException.java
package domain.exceptions;

public class ValidatorException extends RuntimeException {

    /**
     * Constructor for ValidatorException class
     * @param err - error to be thrown
     */
    public ValidatorException(String err){
        super(err);
    }

    /**
     * Constructor with no parameters
     */
    public ValidatorException() {}
}

File: Semester4/Advanced Programming/ThriftProj/C#Thrift/Lab2_Festival/Common/domain/exceptions/ValidatorException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab2_Festival.domain.exceptions
{
    /// <summary>
    /// Class for ValidatorException
    /// </summary>
    class ValidatorException : Exception
    {
        /// <summary>
        /// Constructor for ValidatorException
        /// </summary>
        /// <param name="err">String error to be thrown</param>
        public ValidatorException(string err): base(err)
        {

        }
    }
}
